ibis/tests/all/test_speedcheck.py

```diff
@@ -0,0 +1,26 @@
+from datetime import datetime
+
+import ibis
+import ibis.bigquery as bq
+import ibis.expr.operations as ops
+from ibis.expr.types import TableExpr
+
+
+def test_large_compile():
```

Comment:
Can you move this to `ibis/sq/tests/test_compiler.py`?

Reply:
Please also add a simple doc for this test, sth like:

"""
Test that compiles a large expression tree finishes within reasonable amount of time.
"""

Reply:
Moved!